release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -167,6 +167,13 @@ For more information about this more flexible SR-IOV network deployment during N
 ==== Update to HAProxy 2.6
 With this release, {product-title} updated to HAProxy 2.6.
 
+[id="ocp-4-14-nmstate-ui-console-update"]
+==== NMstate Operator updated in console
+
+With this release, you can access NMstate Operator and resources from the web console. The Kubernetes NMState Operator provides a Kubernetes API for performing state-driven network configuration across the {product-title} cluster’s nodes with NMState.
+
+In the *Administrator* perspective of the console to access the NMState Operator and resources, navigate to *Networking* -> *NodeNetworkConfigurationPolicy*. See xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[About the Kubernetes NMState Operator] for more information and how to install the operator.
```

Comment:
I believe there is another new page in this feature, Networking -> NodeNetworkState, should we also add this information in the RN?
So, for NMState resources **NNCP/NNCE**, navigate to **Networking -> NodeNetworkConfigurationPolicy**, and for NMState resource **NNS**, navigate to **Networking -> NodeNetworkState**.